solutions/CF Good Subarrays.mdx

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
edit [this file](https://github.com/cpinitiative/usaco-guide/blob/master/content/3_Silver/Prefix_Sums.problems.json#L128) to include this internal solution by changing `kind` to `internal` and removing `site`